Basic-Car-Maintenance/Shared/Localizable.xcstrings

```diff
@@ -135,7 +136,7 @@
       }
     },
     "Add" : {
-      "comment" : "Label for submit button on form to add an entry",
+      "comment" : "Label for submit button on form to add an entry\nLabel for adding maintenance event on Dashboard view",
```

Comment:
Note that this one has a `\n` on the comment, which means that this key `Add` has two different string comments, which at the moment causes a conflict.

To fix this you need to either use the same comment for both, or make a new localized string key for `Add` and use your separate comment.

I think making a new one is the best solution

Reply:
Done

Reply:
This wasn't resolved